bin/plpbridge

```diff
@@ -263,9 +271,17 @@ elif args.chip is not None:
 else:
     raise Exception('A chip or a config file must be specified')
 
```

Comment:
This script should remain generic, could you move this part into the flash command in the other python class ?